client/app/(auth)/hooks/index.ts

```diff
@@ -1,2 +1,3 @@
+export { useAccountEdit } from './useAccountEdit'
```

Comment:
This one too! Move to `app/account/hooks`

Reply:
fixed it.
52ad6740cc927e9c3e1817baca316720d513360f